submission.py

```diff
@@ -21,17 +23,43 @@ def _catalogued_action(state, player):
 
 
 def _planned_action(game, state, player):
-    return IterativeDeepening(
-        negamax,
-        timeout=game.timeout * TIMEOUT_BUFFER,
-        max_depth=game.rows * game.columns - state.counter,
-    )(
-        game=game,
-        state=state,
-        player=player,
-        heuristic_func=heuristic,
-        order_actions_func=order_actions,
-    )
+    result = None
+    timeout = game.timeout * TIMEOUT_BUFFER
```

Comment:
you didnt adjust the TIMEOUT_BUFFER in the `config.py`? didnt you say it now runs on a value of 0.975?

Reply:
yeah, the timeout buffer is set to 0.975 in config.py, which was the original value. Is it different for you?